cli/src/command/account.rs

```diff
@@ -644,8 +664,15 @@ pub async fn node_info_command(account: &Account) -> Result<(), Error> {
 }
 
 /// `output` command
-pub async fn output_command(account: &Account, output_id: String) -> Result<(), Error> {
-    let output = account.get_output(&OutputId::from_str(&output_id)?).await;
+pub async fn output_command(account: &Account, selector: OutputSelector) -> Result<(), Error> {
+    let output = match selector {
+        OutputSelector::Id(id) => account.get_output(&id).await,
+        OutputSelector::Index(index) => {
+            let mut outputs = account.outputs(None).await?;
+            outputs.sort_by(|a, b| a.output_id.cmp(&b.output_id));
```

Comment:
Why?

Reply:
Yes

Reply:
If not required I can remove it. Or should we get hold of their creation timestamp to have a similar behavior as for the transactions?

Reply:
Have you tried this? If we sort here but it's not sorted in the actual `outputs_command`, it simply won't work.

Reply:
I see

Reply:
testing it now ... I'll come back when I think it's done.

Reply:
werks